app/services/flexible_translation_service.py

```
@@ -0,0 +1,347 @@
+            },
+            "deepseek": {
+                "models": {
+                    "deepseek-chat": {"cost": "very_low", "quality": "medium", "speed": "fast"},
+                    "deepseek-coder": {"cost": "very_low", "quality": "medium", "speed": "fast"}
+                },
+                "default": "deepseek-chat"
+            }
+        }
+    
+    async def translate(
+        self, 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)